tests/integration/base.py

```diff
@@ -150,15 +151,40 @@ def databricks_cluster_profile(self):
             },
             'test': {
                 'outputs': {
-                    'dbsql': {
+                    'cluster': {
                         'type': 'databricks',
                         'host': os.getenv('DBT_DATABRICKS_HOST_NAME'),
                         'http_path': os.getenv('DBT_DATABRICKS_CLUSTER_HTTP_PATH'),
                         'token': os.getenv('DBT_DATABRICKS_TOKEN'),
                         'schema': self.unique_schema()
                     },
                 },
-                'target': 'dbsql'
+                'target': 'cluster'
+            }
+        }
+
+    def databricks_uc_cluster_profile(self):
+        return {
+            'config': {
+                'send_anonymous_usage_stats': False
+            },
+            'test': {
+                'outputs': {
+                    'uc_cluster': {
+                        'type': 'databricks',
+                        'host': os.getenv('DBT_DATABRICKS_HOST_NAME'),
+                        'http_path': os.getenv('DBT_DATABRICKS_UC_CLUSTER_HTTP_PATH'),
+                        'token': os.getenv('DBT_DATABRICKS_TOKEN'),
+                        'schema': self.unique_schema(),
+                        # TODO: catalog should be set as 'catalog' or 'database'
+                        #       instead of using 'session_properties'
+                        # 'catalog': os.getenv('DBT_DATABRICKS_UC_DEFAULT_CATALOG', 'main'),
+                        'session_properties': {
+                            'databricks.catalog': os.getenv('DBT_DATABRICKS_UC_DEFAULT_CATALOG', 'main'),
```

Comment:
We can set the default catalog by changing this session config? I though we need to use "USE CATALOG main`?